cfr.json

```diff
@@ -1393,6 +1393,12 @@
         "string_id": "cfr-doorhanger-fission-body"
       },
       "icon": "chrome://browser/skin/connection-secure.svg",
+      "info_icon": {
+        "label": {
+          "string_id": "cfr-doorhanger-extension-sumo-link"
+        },
+        "sumo_path": "extensionrecommendations"
```

Comment:
Out of curiosity, what's that for? Fission is not related to extensions and there's nothing on Sumo about Fission.